src/coqLsp/coqLspTypes.ts

```diff
@@ -156,3 +156,13 @@ export class CoqLspError extends Error {
         this.name = "CoqLspError";
     }
 }
+
+export class CoqLspStartupError extends Error {
```

Comment:
Maybe it's reasonable to make `CoqLspStartup` extend `CoqLspError` as a specific case of Coq LSP failure? Or if it does not seem right, I'd suggest renaming `CoqLspError` to something more specific of what it should actually mean.

Reply:
Done. Also moved other extensions of the CoqLspError to the same location.